azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -173,6 +181,16 @@ public FlowRunnerManager(final Props props,
       this.projectDirectory.mkdirs();
     }
 
+    this.projectDirMaxSizeInMB = props.getLong(ConfigurationKeys.PROJECT_DIR_MAX_SIZE,
```

Comment:
"projectDirMaxSizeInMb" see coding standard naming.

Reply:
updated in #1812